pkg/ide/vscode.go

```diff
@@ -51,7 +51,12 @@ func OpenVSCode(activeProfile config.Profile, workspaceId, repoName string, work
 
 func setupVSCodeCustomizations(workspaceHostname string, workspaceProviderMetadata string, tool devcontainer.Tool, codeServerPath string, settingsPath string, lockFileName string) error {
 	// Check if customizations are already set up
-	err := exec.Command("ssh", workspaceHostname, "test", "-f", fmt.Sprintf("$HOME/%s-%s", lockFileName, string(tool))).Run()
+	lockFileNamePath := fmt.Sprintf("$HOME/%s-%s", lockFileName, string(tool))
+	if runtime.GOOS == "windows" {
```

Comment:
@Rutik7066 you're checking if the host OS is windows here but you need to check if the remote OS is windows instead.